ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Shared/_SchoolApplicationFormLayout.cshtml

```diff
@@ -0,0 +1,64 @@
+﻿@using Microsoft.Extensions.Hosting
```

Comment:
I'm not keen on the name of this layout file. It's really a layout for anything that doesn't want to show headers and footers. Maybe something like _LayoutNoHeaderOrFooter ?

Reply:
done